madgraph/iolibs/export_v4.py

```diff
@@ -1738,7 +1738,25 @@ def get_pdf_lines(self, matrix_element, ninitial, subproc_group = False, vector=
         if vector:
             pdf_definition_lines_vec = ""
             pdf_data_lines_vec = ""
-            pdf_lines = " DO iVEC=1,VECSIZE_USED\n"
+            pdf_lines = """ NB_WARP_USED = VECSIZE_USED / WARP_SIZE
```

Comment:
Hi @oliviermattelaer about your question: I am doing nb_warp_used in various places. This is one example here.

But you are right that there may be more. (See https://github.com/madgraph5/madgraph4gpu/issues/983)

Ok I will check dsample.f more in detail...